app.py

```diff
@@ -5,7 +5,7 @@
 
 from composition import composition_server, composition_ui
 from export import export_ui, export_server
-from dgea.dgea import dgea_server, dgea_ui
+from dgea.dgea_1 import dgea_server, dgea_ui
```

Comment:
Why `dgea_1`?

Reply:
I didn't wanted to delete the deseq completely, so I created dgea for deseq and dgea_1 for scvi, so I can change to test and to see if they both produce similar results.

Reply:
Then give it an appropriate name

Reply:
ok